ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -122,19 +122,13 @@ final class Runner[T <: ReadWriteService, Extra](
                     .map(_.start(config.metricsReportingInterval.getSeconds, TimeUnit.SECONDS))
                     .acquire()
                 )
-                ledger <- factory
-                  .readWriteServiceOwner(config, participantConfig, sharedEngine)
+                packageService <- factory
+                  .writePackageOwner(config, participantConfig, sharedEngine)
                   .acquire()
```

Comment:
I wonder if constructing the write service twice is somehow causing uploads to fail. Perhaps there's some shared resource that shouldn't be shared?

Reply:
That's what I'm investigating now as well. Checked that beforehand as well but couldn't identify any problematic usages.